channels/tsmf/client/alsa/tsmf_alsa.c

```diff
@@ -68,8 +68,7 @@ static BOOL tsmf_alsa_open(ITSMFAudioDevice *audio, const char *device)
 	TSMFAlsaAudioDevice *alsa = (TSMFAlsaAudioDevice *) audio;
 	if(!device)
 	{
-		if(!alsa->device[0])
-			strncpy(alsa->device, "default", sizeof(alsa->device));
+		strncpy(alsa->device, "default", sizeof(alsa->device));
 	}
```

Comment:
@akallabeth @bmiklautz this remark is the show stopper for the merge, are you convinced by @ivan-83 's arguments ? If so feel free to merge the PR.